server/public/index.go.html

```diff
@@ -2,21 +2,27 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>{{ __ "offen" }}</title>
-  <link rel="stylesheet" type="text/css" href="/normalize.css">
-  <link rel="stylesheet" type="text/css" href="/skeleton.css">
+  <link rel="stylesheet" type="text/css" href="/tachyons.css">
   {{ with .rootAccount }}
     <script src="/script.js" data-account-id="{{ . }}"></script>
   {{ end }}
 </head>
-<body>
-  <div class="container">
-    <div class="row">
-      <div class="twelve columns">
-        <h1>{{ __ "Welcome to offen" }}</h1>
-        <p>{{ __ "Access your data in the <a href=\"auditorium\">auditorium</a>" }}</p>
+<body style="background-color: #fdfdfd;">
+  <div class="mw7 center pa3 pb5 f5 sans-serif dark-gray">
+    <h1 class="f2 normal mt0 mb4">Welcome to <strong>offen</strong></h1>
+    <div class="w-100 pa3 mt4 mb2 ba b--black-10 br2 bg-white">
+      <div class="mw5 center mt4">
+        <p class="mt0 mb1">View your <strong>user</strong> data.</p>
+        <button class="w-100 f5 link dim bn ph3 pv2 mr1 mb4 dib white bg-gold" data-role="purge" onclick="location.href='auditorium'">
```

Comment:
`data-role="purge"` is probably a left-over from copy/pasting and does not belong here.